future/src/ct/ct_actions_others.cc

```diff
@@ -264,33 +267,39 @@ void CtActions::link_clicked(const Glib::ustring& tag_property_value, bool from_
          Glib::ustring clean_weblink = str::replace(vec[1], "amp;", "");
          if (_pCtMainWin->get_ct_config()->weblinkCustomOn)
          {
-             // todo: subprocess.call(self.weblink_custom_action[1] % clean_weblink, shell=True)
+             std::string cmd = fmt::sprintf(_pCtMainWin->get_ct_config()->weblinkCustomAct, clean_weblink);
+             int retr = std::system(cmd.c_str());
+             if (retr == -1) {
+                 // Internal std::system error
+                 spdlog::error("Error while executing: '{}'; Message: {}", cmd, std::strerror(errno));
+                 return;
+             }
          }
          else g_app_info_launch_default_for_uri(clean_weblink.c_str(), nullptr, nullptr); // todo: ?
      }
      else if (vec[0] == CtConst::LINK_TYPE_FILE) // link to file
      {
-         Glib::ustring filepath = CtExport2Html::_link_process_filepath(vec[1]);
-         if (not Glib::file_test(filepath, Glib::FILE_TEST_IS_REGULAR))
+         fs::path filepath = CtExport2Html::_link_process_filepath(vec[1]).c_str();
+         if (not Glib::file_test(filepath.string(), Glib::FILE_TEST_IS_REGULAR))
          {
-             CtDialogs::error_dialog(str::format(_("The File Link '%s' is Not Valid"), std::string(filepath)), *_pCtMainWin);
+             CtDialogs::error_dialog(fmt::format("The File Link '{}' is Not Valid", filepath), *_pCtMainWin);
              return;
          }
          if (from_wheel)
-             filepath = Glib::path_get_dirname(CtFileSystem::abspath(filepath));
-         CtFileSystem::external_filepath_open(filepath, true);
+             filepath = Glib::path_get_dirname(CtFileSystem::abspath(filepath).string());
+         CtFileSystem::external_filepath_open(filepath, true, _pCtMainWin->get_ct_config());
      }
      else if (vec[0] == CtConst::LINK_TYPE_FOLD) // link to folder
      {
-         Glib::ustring folderpath = CtExport2Html::_link_process_folderpath(vec[1]);
-         if (not Glib::file_test(folderpath, Glib::FILE_TEST_IS_DIR))
+         fs::path folderpath = CtExport2Html::_link_process_folderpath(vec[1]).c_str();
+         if (not fs::is_directory(folderpath))
          {
-             CtDialogs::error_dialog(str::format(_("The Folder Link '%s' is Not Valid"), std::string(folderpath)), *_pCtMainWin);
+             CtDialogs::error_dialog(fmt::format("The Folder Link '{}' is Not Valid", folderpath), *_pCtMainWin);
```

Comment:
You have removed a translatable string here _("translatable string"), is it intentional?